metrics/methelp/register.go

```diff
@@ -0,0 +1,36 @@
+package methelp
+
+import (
+	metrics2 "github.com/VictoriaMetrics/metrics"
+	metrics "github.com/ledgerwatch/erigon/metrics"
+)
+
+func GetOrCreateCounter(s string) *metrics2.Counter {
```

Comment:
I don’t understand what this helpers are doing is needed

Reply:
So I am using this to register all the metrics to DefaultRegistry. I'm using registry to fetch metrics and write it into the http writer. I have avoided writePrometheus function of the VictoriaMetrics because that doesn't add metadata and thus is not OpenMetrics spec compatible.

Reply:
need add some comment with link to this issue and to victoria metrics issue